sklearn/tests/test_multioutput.py

```diff
@@ -13,19 +13,20 @@
 from sklearn.utils.testing import assert_greater
 from sklearn.utils.testing import assert_not_equal
 from sklearn.utils.testing import assert_array_almost_equal
+from sklearn.utils.testing import assert_less
 from sklearn import datasets
 from sklearn.base import clone
 from sklearn.datasets import make_classification
 from sklearn.ensemble import GradientBoostingRegressor, RandomForestClassifier
 from sklearn.exceptions import NotFittedError
 from sklearn.externals.joblib import cpu_count
 from sklearn.linear_model import Lasso
-from sklearn.linear_model import LogisticRegression
+from sklearn.linear_model import LogisticRegression, LinearRegression
```

Comment:
I think we tend to prefer having a single import by line, for merging conflict anticipation